app/src/main/java/com/kickstarter/ui/activities/LoginToutActivity.kt

```diff
@@ -55,10 +58,23 @@ class LoginToutActivity : ComponentActivity() {
             this.ksString = requireNotNull(env.ksString())
             darkModeEnabled =
                 env.featureFlagClient()?.getBoolean(FlagKey.ANDROID_DARK_MODE_ENABLED) ?: false
+            theme = env.sharedPreferences()
+                ?.getInt(SharedPreferenceKey.APP_THEME, AppThemes.MATCH_SYSTEM.ordinal)
+                ?: AppThemes.MATCH_SYSTEM.ordinal
         }
 
         setContent {
-            KickstarterApp(useDarkTheme = if (darkModeEnabled) isSystemInDarkTheme() else false) {
+            KickstarterApp(
+                useDarkTheme =
+                if (darkModeEnabled) {
```

Comment:
Wondering if we could move this piece to some kind of composable extension function that receives the theme shared preference 🤔

Reply:
for sure we can